book/src/data-model/sboms.md

```diff
@@ -0,0 +1,18 @@
+## SBOMs
+
+The `Sbom` entity records that an SBOM was ingested or generated for a `Package` that is being
+_directly monitored_. See the [`Packages`](./packages.md) documentation for a discussion.
```

Comment:
Humm... `directly` monitored, but the term for these SBOMs is `primary` now.  Not sure if there is a change here, but the `primary` term seems to need some exposure here?

Reply:
Good point. I haven't written the `Package` page of the data-model yet. Seems like that is the right place no?

Reply:
Yeah, it totally does to me.  Please!